src/process_forest.py

```diff
@@ -213,7 +218,10 @@ def analyze(self, entries):
             if entry.is_process_created_event() or entry.is_sysmon_proc_created_event():
                 process = entry.get_process_from_event()
                 if process.pid in open_processes:
-                    self._logger.warning("collision on pid: %x", process.pid)
+                    if args.hexpids:
+                        self._logger.warning("collision on pid: %x", process.pid)
+                    else:
+                        self._logger.warning("collision on pid: %d", process.pid)
```

Comment:
A bit of duplication here. Maybe should format the pid into a temporary variable, and then splice that into the log message?